charts/astronomer/values.yaml

```diff
@@ -423,6 +423,7 @@ registry:
   logLevel: info
 
 install:
+  enabled: true
```

Comment:
`.Values.install.enabled` is not conveying the reference to CLI 
We should rename the flag to `cliEnabled` for better clarity

Reply:
this might affect existing deployment config since customers are using this for a long time we can add both configs and we can state old config will be removed in 0.34

Reply:
i agree with @rishkarajgi that having cliInstall would have been better, but i also agree with @pgvishnuram that changing the parent namespace of the new `enabled` parameter is outside of the scope of this ticket and could introduce problems. if we want to improve the `install` value namespace, we should do that on a minor version bump and address that in another ticket.

Reply:
i experienced this ambiguity again today when looking at [the missing probes issue](https://github.com/astronomer/issues/issues/6606) and yeah, it's too ambiguous. however, i think we should just get rid of cli-install altogether. <https://github.com/astronomer/issues/issues/6611>